sandbox/grommet-app/src/pages/layouts/kinds/HPEMarketing/HPEAIUseCase.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useContext } from 'react';
+import {
+  Box,
+  Button,
+  Grid,
+  Heading,
+  Image,
+  PageContent,
+  ResponsiveContext,
+} from 'grommet';
+import { LinkNext } from 'grommet-icons';
+import { CardTemplate } from './components/CardTemplate';
+
+export const HPEAIUseCase = () => {
+  const breakpoint = useContext(ResponsiveContext);
+
+  const columns = ['xsmall', 'small'].includes(breakpoint)
+    ? ['auto']
+    : { count: 3, size: 'auto' };
+  const rows = ['auto'];
+  const gap = {
+    medium: 'medium',
+    large: 'large',
+    xlarge: 'large',
+  };
+
+  return (
+    <PageContent
+      pad={{
+        top: ['xsmall'].includes(breakpoint) ? 'xlarge' : 'none',
+        bottom: 'xlarge',
+        horizontal: 'xlarge',
+      }}
+      background={{
+        fill: 'horizontal',
+        image: 'url(/video-marketing-background.png)',
+        dark: false,
+      }}
+    >
+      <Box gap="medium">
+        {/* in marketing page they have 60px heading
+        even with size xlarge we are at 48px  */}
+        <Heading size="xlarge" margin="none" level={2}>
+          AI use cases
+        </Heading>
+        <Grid columns={columns} rows={rows} gap={gap[breakpoint]}>
+          <CardTemplate
+            pad={null}
+            border={false}
+            media={
+              <Image
+                src="/VirtualAssistant.png"
+                alt="hpe cray supercomputing"
+                fit="cover"
+                // this is not ideal
+                style={{
+                  borderBottomLeftRadius: '12px',
+                  borderBottomRightRadius: '12px',
+                }}
+              />
+            }
+            title="HPE Cray Supercomputing"
+            description="Exascale technologies to unlock the next frontier of discovery, innovation, and achievement."
+            actions={
+              // button sizes are off in design they have 42px height and we have 36px
+              <Button
+                label="View the usecase brief"
```

Comment:
Same comment as above about adding aria-label that also includes title. For example "View the usecase brief for [title]"